### PR TITLE
css: make the menu / header more compact

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by psychon** [comment]
Dunno how this looks like, so fine with me, I guess.
Due to "recent things", @aajjbb might be the right person to ask. ;-)

**Comment by blueyed** [comment]
Pushed some more commits, containing 399b83e to fix the menu being underlined actually.

File: local.css [hunk 1]
```diff
@@ -316,3 +316,5 @@ blockquote{
 #donate p, #donate div {
   text-align: center;
 }
+
+/* vim: set sw=2 et: */
```
Comment (psychon) [inline local.css]: Thanks :-)

**Comment by aajjbb** [comment]
The changes are fine for me, the CSS linter is a great idea, as well as the smaller menu.

I'm sorry for the tripled center-logo thing. I interactively rebased commits for the first time, so mistakes we 'expected'.

**Pull request merged**